.github/workflows/kuksa_databroker_build.yml

```diff
@@ -126,26 +115,28 @@ jobs:
       working-directory: ${{github.workspace}}/
       run: |
         which cross || cargo install cross
+        cargo install cargo-license cargo-cyclonedx
+        pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@6f3d7627760582d8ba83cc8a0f7449d00fffee84#subdirectory=sbom-tools"
```

Comment:
Would it be a nice to long term ambition to publish "kuksa-sbom-tools" as pypi package ?

Reply:
yes. Or at least tag it. But for now, it being an internal tools that might still change, I think using hash is ok